gradle.properties

```diff
@@ -4,13 +4,14 @@ micronautVersion=3.1.4
 micronautTestVersion=3.0.3
 groovyVersion=3.0.9
 spockVersion=2.0-groovy-3.0
-postmarkVersion=
 title=Micronaut Email
 projectDesc=Integration with Transaction Email Providers
 projectUrl=https://micronaut.io
 githubSlug=micronaut-projects/micronaut-email
 developers=Graeme Rocher
-
+sendgridVersion=4.8.0
+postmarkVersion=1.7.5
+mailjetVersion=5.2.0
```

Comment:
Add them to `bomProperties`

Reply:
There's been some new updates of libraries:

- a [4.8.2 version](https://search.maven.org/artifact/com.sendgrid/sendgrid-java) of sendgrid-java now 
- a [1.8.0 version](https://search.maven.org/artifact/com.wildbit.java/postmark) of postmark
- and a [5.2.0 version](https://search.maven.org/artifact/org.apache.poi/poi-ooxml) of poi-ooxml

Do we want to use the latest?

Reply:
Given the discussion in the last call this project needs a BOM and doesn't need to add them to `bomProperties`